### PR TITLE
stack.stack version 3.30.4

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15966)

### CODE REVIEW

File: manifests/s/stack/stack/3.30.4/stack.stack.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Tags:
+- browser
+- electron
+- chrome
+- webapp
+InstallerType: exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://binaries.getstack.app/builds/prod/win/x64/Stack%20Setup%203.30.4-x64.exe
+  InstallerSha256: f046633632fe90e6bb854de210c4583c62eb32d9e0a85ccb88506806591f2c98
```
Comment (jedieaston) [inline manifests/s/stack/stack/3.30.4/stack.stack.yaml:21]:
```suggestion
  InstallerSha256: f046633632fe90e6bb854de210c4583c62eb32d9e0a85ccb88506806591f2c98
  InstallerSwitches: 
      Silent: " "
      SilentWithProgress: " "
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Can you add these empty switches to get rid of the warning? Thanks!

File: manifests/s/stack/stack/3.30.4/stack.stack.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate
+
+PackageIdentifier: stack.stack
+PackageVersion: 3.30.4
+PackageLocale: en-US
+Publisher: Stack
```
Comment (ItzLevvie) [inline manifests/s/stack/stack/3.30.4/stack.stack.yaml:6]:
```suggestion
Publisher: Stackers INC
```
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/stack/stack/3.30.4/stack.stack.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate
+
+PackageIdentifier: stack.stack
+PackageVersion: 3.30.4
+PackageLocale: en-US
+Publisher: Stack
+Author: Stack
+PackageName: Stack
+PackageUrl: https://stackbrowser.com/
+License: copyright (c) stack
```
Comment (ItzLevvie) [inline manifests/s/stack/stack/3.30.4/stack.stack.yaml:10]:
```suggestion
License: Copyright (c) Stackers INC.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]
image

**Pull request merged**